frontend/micro-ui/web/micro-ui-internals/packages/modules/dss/src/pages/Home.js

```
@@ -107,7 +107,20 @@ const Chart = ({ data, moduleLevel, overview = false }) => {
-          {insight?.[0] && `${insight[0]}% ${t(Digit.Utils.locale.getTransformedLocale("DSS" + insight?.[1] || ""))}`}
+          {/* {insight?.[0] ? `${insight[0]} ${t(Digit.Utils.locale.getTransformedLocale("DSS" +  (insight?.[1] ?? "")))}` : ""} */}
+          {/* {insight?.[0] !== undefined && insight?.[0] !== ""
+            ? (isNaN(insight[0])
+              ? `${insight[0]}` + ` ${t(Digit.Utils.locale.getTransformedLocale("DSS" + (insight?.[1] ?? "")))}`
+              : `${insight[0]}%`
+            )
+            : ""} */}
+           {
+             insight && insight[0] !== undefined && insight[0] !== "" 
+             ? isNaN(insight[0]) 
+               ? `${insight[0]} ${t(Digit.Utils.locale.getTransformedLocale("DSS" + (insight[1] ? insight[1] : "")))}`
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)